conf/chesapeake_cvpr.yaml

```diff
@@ -0,0 +1,22 @@
+trainer:
```

Comment:
My biggest complaint about all this OmegaConf stuff is that there doesn't seem to be any documentation on what options are supported or what possible values they can take. There's no way to get a help message without argparse.

Reply:
This is discussed in much more detail here -- https://github.com/facebookresearch/hydra/issues/633.

For now I think comments in the yaml files are OK -- we can do more here. I think there will be few scenarios in which a user is trying to configure experiments without looking at the trainer code.